src/main/java/com/iexec/common/replicate/ReplicateStatusUpdate.java

```diff
@@ -76,11 +76,13 @@ public boolean isSuccess() {
 
     @JsonIgnore
     public ReplicateStatusDetails getDetailsWithoutStdout() {
```

Comment:
`public ReplicateStatusDetails getDetailsWithoutLogs()`
?
The idea is to avoid printing `details.toString()` in a small `[details:{}]`